extensions/PhabBugz/lib/Util.pm

```diff
@@ -80,6 +80,22 @@ sub set_attachment_approval_flags {
       . $approval_flag_name
       . $status);
 
+  # If we are setting to + or -, we need to make sure
+  # Phabricator user is a member of the release-managers project
+  my $is_release_manager = 0;
+  my $release_manager_project
+    = Bugzilla::Extension::PhabBugz::Project->new_from_query(
+    {name => 'release-managers'});
+  if ($release_manager_project) {
```

Comment:
Nit: this section of code to check if a user is a release manager could be a function.

Reply:
I will add a method to the Phab User class that checks for release manager.